benchmarks/performance_test.py

```diff
@@ -1,17 +1,78 @@
 from urlparse4 import urlsplit, urljoin
 from timeit import default_timer as timer
 
-total = 0
+from sys import argv
 
-with open('urls/chromiumUrls.txt') as f:
-    for url in f:
+encode = False
 
-        start = timer()
+try:
+    if argv[1] == "encode":
+        encode = True
+except IndexError:
+    print("encode is not defined, continue with the program...")
 
-        a = urlsplit(url)
+if encode:
+    total = 0
 
-        end = timer()
+    for i in range(5):
+        with open('urls/chromiumUrls.txt') as f:
+            for url in f:
 
-        total += end - start
+                start = timer()
 
-print("the total time is", total, "seconds")
+                a = urlsplit(url.encode())
```

Comment:
I think it's better to move encoding of url outside of timed part, and specify encoding